docs/Microservices/RPC/buildRPC_Manual.md

```diff
@@ -0,0 +1,228 @@
+In addition to using GitHub Actions to obtain a binary, the RPC binary can also be manually built. The steps below walk through how to build RPC on Windows 10, Ubuntu (18.04 or 20.04), and CentOS7/8.
+
+### Required Software
+
+- [git](https://git-scm.com/downloads)
+
+Additionally, if using Windows 10:
+
+- [Microsoft Visual Studio*](https://visualstudio.microsoft.com/): 2019 or newer version of Visual Studio Community/Professional
+    - Make sure to install the **Desktop development with C++** package at time of installation or via the 'Get tools and extensions' menu within Microsoft Visual Studio*.
+
+<br>
+
+The steps below assume the following directory structure where rpc is the clone of the [rpc repository](https://github.com/open-amt-cloud-toolkit/rpc), vcpkg is a clone of the VCPKG tool source and build is the RPC build directory. Both vcpkg and build directories will be created in later steps.
+
+```
+\rpc
+  |__vcpkg
+  |__build
+```
+
+
+### Clone the Repository
+
+1. On your development system, navigate to a directory of your choice to clone and build RPC.
+
+2. Clone the RPC repository.
+
+    ``` bash
+    git clone https://github.com/open-amt-cloud-toolkit/rpc.git && cd rpc
+    ```
+
+### Install Prerequisites and Build RPC
+
+=== "Windows"
+    Open 'x64 Native Tools Command Prompt for VS 20XX' as Administrator on your development system.  **This is NOT a regular Windows Command Prompt.**  This specific tool is used for compiling the RPC executable.
+    
+    ![NTCP](../../assets/images/x64NativeToolsCP.png)
+
+    **Build VCPKG and C++ REST SDK**
+
+
+    1. In the `rpc` directory, clone the VCPKG repository. Vcpkg is a C/C++ Library Manager for Windows that was created by Microsoft.  Find out more about it [here](https://github.com/microsoft/vcpkg).
+    ``` bash
+    git clone --branch 2020.11-1 https://github.com/microsoft/vcpkg.git && cd vcpkg
+    ```
+
+    2. Build vcpkg.exe using the following command.
+    ``` bash
+    bootstrap-vcpkg.bat
+    ```
+        
+    3. Install C++ REST SDK. This can take anywhere from 8 - 15 minutes depending on download speeds and installation times.
+    ``` bash
+    vcpkg install cpprestsdk[websockets]:x64-windows-static
+    ```
+
+    **Build RPC**
+
+    1. Return to the `rpc` directory and create a new `build` directory.
+    ``` bash
+    cd .. && mkdir build && cd build
+    ```
+    
+    2. Generate the CMake config.
+    ``` bash
+    cmake -DVCPKG_TARGET_TRIPLET=x64-windows-static -DCMAKE_TOOLCHAIN_FILE=../vcpkg/scripts/buildsystems/vcpkg.cmake ..
```

Comment:
I've had trouble with relative paths in the -DCMAKE_TOOLCHAIN_FILE. If this works, no problem. Just pointing out my experience.

Reply:
I'd be curious with other's experiences, because I ran into the inverse and that is why I switched them.  On both my Win10 and Ubuntu20.04 NUCs, the cmd would error and refuse to run if I supplied /rpc/vcpkg rather than ../vcpkg

Reply:
The full path should always work. But, as long as we verified relative paths work as well, fine with me!